### ISSUE TITLE
Prevent loading of older videos

### ISSUE THREAD

**Issue by OsaSoft** [OsaSoft/youtube-better-subscriptions#63]
Another nice option to be able to configure, is to prevent the page loading videos older than a certain age (think Plex's On Deck feature if you have used Plex. You can specify an age of videos to appear On Deck before they get removed automatically).

Reasoning behind this:
My goal is to have an empty feed so I know I am 100% up to date.
If I mark all videos as watched, the page will go blank, but then starting loading older videos.
You can mark all as watched again, but it will then load even older videos. I could keep doing this of course, but would have to do it so many times to mark all of my years worth of videos as watched it's not a feasible option.

It would be great to prevent YouTube loading any additional videos past a certain age, so that a blank feed can be maintained.

_Originally posted by @Eds1989 in https://github.com/OsaSoft/youtube-better-subscriptions/issues/51#issuecomment-439611981_

**Comment by Nadastorm** [comment]
+1

I'm having the same issue since I just marked all videos as watched.

**Comment by ms411** [comment]
It is genuinely quite annoying that it constantly tries to pull videos that are months even years old that I've already watched, refreshing the feed over and over. There should be a configurable threshold.

**Comment by OsaSoft** [comment]
related to #82

**Comment by Galileji** [comment]
This feature would make a lot of sense!

**Comment by bonbonboi** [comment]
Yes indeed, an option to limit the older videos loading since a specific date or an option like "don't load videos older than a week or a month" and so on, that's because when I go to my YT subscription page the extension keeps looking for videos and it doesnt give up (timeout) while that is going on, the cpu usage is high, the extension is trying to find those older videos to display, like 7 months ago videos and older than that, I don't need those too old videos, if I do then I can go directly to a channel page and look for those old videos there. 

I'm going to answer myself here as a thought: 
(I think that option won't change anything regarding to the performance, since the extension needs to load every video in order to read its date then hide it if its date is older than x, so we back to the rabbit hole again? because it looks like the "view" status, the extension needs to load all the feed in order to know  the "view" status, if seen, it hides it if not then it keep it in the feed)

**Comment by tummyslyunopened** [comment]
Looking for feedback on #191 all testers welcome.
